# 分别给出飞机的前后左右离障碍物的距离
std_msgs/Header header

float32 front_dist
float32 left_dist
float32 behind_dist
float32 right_dist

